### ISSUE TITLE
Something is wrong with lobbyID's

### ISSUE THREAD

**Issue by zondor** [RJacksonm1/node-dota2#65]
Hi to @RJacksonm1 , @paralin 
What was the goal : create small service for friends ,with bot, where i can send to bot some lobby match params ( like game type, win/ loss numbers) and then invite friends to this lobby. 

Results :
When bot is started, i used "practiceLobbyCreateResponse" to get the id . But with this id, all other requests like "configPracticeLobby", "joinPracticeLobby", or launch - returns  DOTA_JOIN_RESULT_INVALID_LOBBY

Tried to get list of lobbies "practiceLobbyListRequest", and  in responce can't find this lobby. 
some samples: 
```
var lobbyOptions = {
    game_name: "PVDraftLobby",
    server_region: 0,
    game_mode: 11,
    allow_cheats: false,
    fill_with_bots: true,
    allow_spectating: true,
    pass_key: "00000000000003",
    series_type: 1,
    radiant_series_wins: 1,
    dire_series_wins: 1,
    allchat: true
};

//--------------------------
            self.Dota2.on('practiceLobbyCreateResponse', function(res, id) {
                if (this.debug) util.log("##Received  New Lobby ID " + id);
                var lobbyId = id;

                self.Dota2.configPracticeLobby(lobbyId, lobbyOptions, function(err, body) {
                //     console.log('#configPracticeLobby');
                //     console.log('##Error');
                //     console.log(err);
                //     console.log('##Body');
                //     console.log(JSON.stringify(body));
                });
            });

// Some log
#configPracticeLobby
##Error
DOTA_JOIN_RESULT_INVALID_LOBBY
##Body
{"result":"DOTA_JOIN_RESULT_INVALID_LOBBY"}

```
Is that an issue, or i'm doing something wrong. 

Is there other posibilite to config the match and provide an steam://IP:PORT  to the frends to join? 
(working on Ubuntu 14 with node 0.12 ) 
Thanks

**Comment by paralin** [comment]
Please see the other issue about this. I've fully mapped out the lobby and party cache systems and what remains in this repository is a hack at best and needs to be rewritten. See paralin/dota2

**Comment by paralin** [comment]
Also you'll want to create a lobby and send the config with the create request not join an existing one. You'll not need to touch lobby ids at all. Make sure you send leave lobby just before the create request to be sure that you aren't already in one.

**Comment by RJacksonm1** [comment]
Should be resolved with #66
